src/bin/tikv-server.rs

```diff
@@ -654,15 +654,16 @@ fn build_raftkv(config: &toml::Value,
                 -> (Node<RpcClient>, Storage, ServerRaftStoreRouter, SnapManager, Arc<DB>) {
     let trans = ServerTransport::new(ch);
     let path = Path::new(&cfg.storage.path).to_path_buf();
-    let opts = get_rocksdb_db_option(config);
-    let cfs_opts = vec![get_rocksdb_default_cf_option(config, total_mem),
-                        get_rocksdb_lock_cf_option(config),
-                        get_rocksdb_write_cf_option(config, total_mem),
-                        get_rocksdb_raftlog_cf_option(config, total_mem)];
+    let db_opts = get_rocksdb_db_option(config);
+    let mut cfs_opts = HashMap::default();
+    cfs_opts.insert(CF_DEFAULT, get_rocksdb_default_cf_option(config, total_mem));
```

Comment:
We may need to consider save the cf opt in rocksdb later, otherwise tikv may core.